Source/UserManagement/Read/DataCollectors/Migration/RemoveNationalSociety.cs

```diff
@@ -0,0 +1,19 @@
+using Infrastructure.Read.Migration;
+
+namespace Read.DataCollectors.Migration
+{
```

Comment:
Why do we need migration on a read model? Why not just remove property?

Reply:
We might have changed the name of field that may or may not be of value to us, we might want to adopt a migration strategy in that case. 

But when we have an event store in place we could just rewind the events and update the read models in that way

Reply:
Sounds good. When the event store and replay is available, this scenario will not be necessary longer, although we can still solve it like this when that is easiest/best